hcl/hcl.go

```diff
@@ -31,40 +31,46 @@ type Module struct {
 	Source string // Source is the module source path (eg.: directory, git path, etc).
 }
 
+type Config struct {
+	Terramate *Terramate
+	Stack     *Stack
+}
+
+// Terramate is the parsed "terramate" HCL block.
 type Terramate struct {
 	// RequiredVersion contains the terramate version required by the stack.
 	RequiredVersion string
 
 	Backend *hclsyntax.Block
 }
 
-// Parser is a terramate parser.
-type Parser struct {
-	p *hclparse.Parser
+// Stack is the parsed "stack" HCL block.
+type Stack struct {
+	// Name of the stack
+	Name string
+
+	// After is a list of non-duplicated stack entries that must run after the
+	// current stack runs.
+	After []string
```

Comment:
This is related to ordering, shouldn't it be only on the order of exec PR ?

Reply:
oh gosh